ring/main.go

```diff
@@ -139,6 +144,9 @@ func helpCmd(args []string) error {
             the number of minutes to wait before reassigning a given replica of
             a partition. This is to give time for actual data to rebalance in
             the system before changing where it is assigned again.
+        configfile=<value>
+            The <value> is the path to a json config file that will be byte encoded
```

Comment:
Note to remove the json reference, since it could be yaml, toml, harryl, whatever.